### PR TITLE
Update JsonTree light theme to have better contrast

### LINKED ISSUE #6321: Slice 3.2: Fix Color Contrast
https://www.notion.so/pixiebrix/Make-Variable-Popover-Generally-Available-e64deef53c894457901e88a606262f04?pvs=4#595ecb7cb2b24de98bf406e680961950

### PR DESCRIPTION
**Pull request by turbochef**
## What does this PR do?

- Closes #6321
- Updates the color of the values and count to be `#007124`
- Updates color of null and boolean values to be `#B4183F`
- Stop nested count color change when expanded

## Demo

Kapture 2023-09-14 at 19 21 38

## Checklist

- [ ] Add tests
- [x] Designate a primary reviewer: @grahamlangford

### CODE REVIEW

File: src/components/fields/schemaFields/widgets/varPopup/VariablesTree.tsx [hunk 1]
```diff
@@ -102,7 +102,6 @@ const VariablesTree: React.FunctionComponent<{
     theme={popoverTheme}
     postprocessValue={sortVarMapKeys}
     shouldExpandNodeInitially={expandCurrentVariableLevel(vars, likelyVariable)}
-    invertTheme
```
Comment (turbochef) [inline src/components/fields/schemaFields/widgets/varPopup/VariablesTree.tsx:105]: Inverting the theme calculated the colors so we lost control of them. I just set the background to white in the light theme.

File: src/components/fields/schemaFields/widgets/varPopup/popoverTheme.ts [hunk 2]
```diff
@@ -40,6 +40,9 @@ export const popoverTheme: Theme = {
     alignItems: "center",
     flexWrap: "wrap",
   },
+  nestedNodeItemString: {
+    color: "#007124",
```
Comment (grahamlangford) [inline src/components/fields/schemaFields/widgets/varPopup/popoverTheme.ts:44]: Can we add this to `src/themes/colors.scss` and use the variable?

File: src/components/jsonTree/JsonTree.tsx [hunk 3]
```diff
@@ -142,6 +142,10 @@ export function searchData(query: string, data: unknown): unknown {
 
 const jsonTreeTheme: Theme = {
   extend: theme,
+  nestedNodeItemString: {
+    // Hard-coded to avoid the count greying out when selected
+    color: "#007124",
```
Comment (grahamlangford) [inline src/components/jsonTree/JsonTree.tsx:147]: https://github.com/pixiebrix/pixiebrix-extension/pull/6448/files#r1327615164

File: src/components/fields/schemaFields/widgets/varPopup/popoverTheme.ts (hunk 2 above)
Comment (grahamlangford) [inline src/components/fields/schemaFields/widgets/varPopup/popoverTheme.ts:44]: NIT: I would prefer to avoid this magic string.

**Review by grahamlangford — APPROVED** [review]

File: src/components/fields/schemaFields/widgets/varPopup/popoverTheme.ts (hunk 2 above)
Comment (turbochef) [reply src/components/fields/schemaFields/widgets/varPopup/popoverTheme.ts]: I changed it to reference the `light.ts` theme.

**Pull request merged**